roles/opensrp/defaults/main.yml

```diff
@@ -68,3 +68,5 @@ opensrp_authencation_cache_ttl: 600
 opensrp_postgres_hba_config_host: "127.0.0.1/32"
 opensrp_install_postgres: yes
 opensrp_postgres_host: "localhost"
+mysql_setup_database_backups: true
```

Comment:
Since MySQL is also used by the OpenMRS role, do you think it'd be better to have this in the mysql role?